agent/agent.go

```diff
@@ -659,7 +659,7 @@ func (a *Agent) connectOutputs(ctx context.Context) error {
 				return err
 			}
 		}
-		log.Printf("D! [agent] Successfully connected to output: %s\n", output.Name)
+		log.Printf("D! [agent] Successfully connected to %s", output.LogName())
```

Comment:
We are a little inconsistent with `[%s]` vs `%s` and LogName, it's a bigger issue to really fixup the log messages but for now lets go with `%s` with no brackets.  I feel like `[agent] [inputs.exec]` is too brackety for my taste.  I also want to get people away from thinking of the plugin as `[inputs.foo]` as it causes some confusion with the toml.

Reply:
Not sure what the recommendation is here. To me, the bracketed thing is the part of telegraf doing the logging (allowing to later filter on the source of the logs (agent, inputs, outputs, etc...)) which is why when one part, in this case agent, logs something about a plugin, that plugin name/alias is not bracketed.